external_libs/mbedTLS/README.txt

```diff
@@ -3,3 +3,7 @@
 # You'll need to download mbedTLS from the official ARMmbed repository and
 # place the files here. We recommend that you pick the latest version of 2.16
 # LTS release in order to have up-to-date security fixes.
+
+# Note that this SDK will not negotiate the TLS Maximum Fragment Length even if
+# it is enabled in mbedTLS' configuration. Therefore, you should ensure content
+# buffers used by mbedTLS are at least 16384 bytes in length.
```

Comment:
May be, providing more context about the reason of suggesting the size will be helpful. (Feel free to re-word my suggestion for something more appropriate)
```suggestion
# buffers used by mbedTLS are at least 16384 bytes in length to use the largest maximum fragment length supported by TLS.
```

Reply:
I like "record length" slightly more so I'll go with that